components/Table/style.module.scss

```diff
@@ -300,3 +300,12 @@ $cell-height: 49px;
     min-width: 80px;
   }
 }
+
+/* Headless Table */
```

Comment:
Let's update it to Headerless, for consistency. I also noticed when I removed the `border-collapse`, it affected the size, and this selector applies to the table in all of its forms, so this is altering the table quite a bit. I see that it doesn't appear without it though. Honestly, I am not sure why, but the table is designed a particular way since it uses CSS for borders (ex: 1px border becomes a 2px border when cells are adjacent), so the table may be intentionally tucked up 1px or something... Regardless, I do see that the `<td>` is receptive to border-top and they already contain other border properties for the table, so perhaps we can try targeting the `td`s on the first row.